testutil/keeper/tokenomics.go

```diff
@@ -81,14 +84,16 @@ type TokenomicsModuleKeepers struct {
 type tokenomicsModuleKeepersConfig struct {
 	tokenLogicModules []tlm.TokenLogicModule
 	initKeepersFns    []func(context.Context, *TokenomicsModuleKeepers) context.Context
-
 	// moduleParams is a map of module names to their respective module parameters.
 	// This is used to set the initial module parameters in the keeper.
 	moduleParams map[string]cosmostypes.Msg
 
 	// proposerConsAddr and proposerValOperatorAddr are used to configure the block proposer
 	proposerConsAddr        cosmostypes.ConsAddress
 	proposerValOperatorAddr cosmostypes.ValAddress
+
+	// validators allows specifying custom validators for testing multi-validator scenarios
+	validators []stakingtypes.Validator
```

Comment:
Having this WHILE keeping "proposer" is a bit confusing.

Should we remove and/or create TECHDEBT??